specification/maintenance/resource-manager/Microsoft.Maintenance/preview/2021-04-01-preview/Maintenance.json

```diff
@@ -0,0 +1,2373 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-04-01-preview",
+    "title": "MaintenanceManagementClient",
+    "description": "Azure Maintenance Management Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Maintenance/publicMaintenanceConfigurations": {
+      "get": {
+        "tags": [
+          "PublicMaintenanceConfigurations"
+        ],
+        "summary": "Get Public Maintenance Configuration records",
+        "operationId": "PublicMaintenanceConfigurations_List",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ListMaintenanceConfigurationsResult"
+            }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/MaintenanceError"
+            }
+          }
+        },
+        "x-ms-pageable": {
```

Comment:
Does your service provides pageable results? If so, please fill in nextLink, otherwise can remove this section.

Reply:
We don't have pageable result but i think it was introduce to suppress arm warning

e.g warning after removing x-ms-pagable

```
WARNING (PageableOperation/R2029/SDKViolation): Based on the response model schema, operation 'Updates_List' might be pageable. Consider adding the x-ms-pageable extension.
    - file:///E:/Github/ARM/azure-rest-api-specs-pr/specification/maintenance/resource-manager/Microsoft.Maintenance/preview/2021-04-01-preview/Maintenance.json:1705:6 ($.paths["/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{providerName}/{resourceType}/{resourceName}/providers/Microsoft.Maintenance/updates"].get)
```